libs/blocks/share/share.js

```diff
@@ -0,0 +1,87 @@
+import { createTag, getConfig } from '../../utils/utils.js';
+
+/**
+ * @param {String} path the path to the SVG
+ * @param {Array} selectors an array of selectors to look for
+ * @returns {Array} an array of objects with
+ */
+export async function getSVGsfromFile(path, selectors) {
+  if (!path) return null;
+  const resp = await fetch(path);
+  if (!resp.ok) return null;
+
+  const text = await resp.text();
+  const parser = new DOMParser();
+  const doc = parser.parseFromString(text, 'image/svg+xml');
+
+  if (!selectors) {
+    const svg = doc.querySelector('svg');
+    if (svg) return [{ svg }];
+    return null;
+  }
+
+  return selectors.map((selector) => {
+    const symbol = doc.querySelector(`#${selector}`);
+    const svg = document.createElementNS('http://www.w3.org/2000/svg', 'svg');
+    while (symbol.firstChild) svg.appendChild(symbol.firstChild);
+    for (let i = symbol.attributes.length - 1; i >= 0; i -= 1) {
+      svg.attributes.setNamedItem(symbol.attributes[i].cloneNode());
+    }
```

Comment:
Does this have to iterate backwards?  I think something like below should work?

```suggestion
[...symbol.attributes].forEach((attr) => s.attributes.setNamedItem(attr.cloneNode()))
```